rundeckapp/grails-app/services/rundeck/services/ReportService.groovy

```diff
@@ -560,4 +563,9 @@ class ReportService  {
 
         return frameworkService.authorizeProjectResources(authContext,resHS, constraints, project)
     }
+
+    private boolean isOracleDatasource(){
```

Comment:
Did you check if this technique works with JNDI data sources? Some of our Oracle clients use JNDI.